cfgov/transition_utilities/tests/test_conditional_urls.py

```diff
@@ -0,0 +1,48 @@
+import mock
+
+from django.test import TestCase, override_settings
+
+from transition_utilities.conditional_urls import include_if_app_enabled
+
+
+class ConditionalURLTests(TestCase):
+
+    @override_settings(
+        LEGACY_APP_URLS={'core': True},
+        INSTALLED_APPS=['core'],
+    )
+    @mock.patch('transition_utilities.conditional_urls.include')
+    def test_include_if_app_enabled_calls_include(self, mock_include):
+        """ Test that an app gets include()ed when all conditions are met """
+        include_if_app_enabled('core', None)
+        self.assertEqual(mock_include.call_count, 1)
+
+    @override_settings(
+        LEGACY_APP_URLS={'core': False},
+        INSTALLED_APPS=['core'],
+    )
+    @mock.patch('transition_utilities.conditional_urls.wagtail_fail_through')
+    def test_include_if_app_enabled_false_in_legacy_apps(
+            self, mock_wagtail_fail_through):
+        """ Test fail-through if LEGACY_APP_URLS[app] is False """
```

Comment:
Maybe I just don't understand the underlying code very well, but this docstring makes little sense to me :\  What is `fail_through` supposed to be, and if we're testing its result why are we mocking it?